exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/parser/CompoundIdentifierConverter.java

```diff
@@ -161,6 +171,7 @@ public SqlNode visitChild(
     rules.put(SqlJoin.class, R(D, D, D, D, D, E));
     rules.put(SqlOrderBy.class, R(D, E, D, D));
     rules.put(SqlDropTable.class, R(D));
+    rules.put(SqlSetOption.class, R(D, D, D));
```

Comment:
There is already a comment explaining the rules, and why they are needed. What more?